### PR TITLE
[10.0][T881][IMP] -  Reformat Code for PCI-Custom Branch

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
Task #[881](https://www.quartile.co/web#id=881&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by TimLai125** [comment]
@YogeshMahera-SerpentCS FYI, please remember to add OCA modules to the exclude list to the .pre-commit-config.yaml

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 @yostashiro For this PR need some repo access of Enterprise module, pci-private repo, for the modules `l10n_us_check_printing_adj` which is depends on the enterprise `l10n_us_check_printing` also one of the modules require `pr1_multi_website` dependency which is on PCI-private

for more info can you please check out the Travis the build [323003250](https://travis-ci.com/github/qrtl/pci-custom/jobs/323003250) and [323027506](https://travis-ci.com/github/qrtl/pci-custom/jobs/323027506)

File: .flake8 [hunk 1]
```diff
@@ -7,4 +7,4 @@ select = C,E,F,W,B,B9
 # E203: whitespace before ':' (black behaviour)
 # E501: flake8 line length (covered by bugbear B950)
 # W503: line break before binary operator (black behaviour)
-ignore = E203,E501,W503
+ignore = E203,E501,W503,F402
```
Comment (TimLai125) [inline .flake8]: Why we are ignoring this? which module casuing this error?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -96,12 +80,3 @@ repos:
...
         exclude: /__init__\.py$
-  - repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: v2.5.0
-    hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
-      - id: check-json
-        files: \.(bowerrc|jshintrc|json)$
-      - id: check-yaml
-      - id: debug-statements
```
Comment (TimLai125) [inline .pre-commit-config.yaml:107]: Why we are removing this part?

Comment (YogeshMahera-SerpentCS) [reply .pre-commit-config.yaml:107]: Added to the above repo with same function

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 in the last commit the In Travis the linting is a pass but the install and test is failing due to  enterprise clone
`subprocess.CalledProcessError: Command '['git', 'clone', '-q', 'https://github.com/odoo/enterprise.git', '-b', '10.00', '--single-branch', '--depth=1', '/home/travis/dependencies/enterprise']' returned non-zero exit status 128`

File: .flake8 (hunk 1 above)
Comment (YogeshMahera-SerpentCS) [reply .flake8]: yes Removed the ignore F402
before
`path = _
for _ in xrange(7):`

after
`path = _
for path in xrange(7):`

**Pull request closed without merging**